PatientRegistrationApp/src/components/SinglePatientReviewSubmit.vue

```diff
@@ -1,269 +1,143 @@
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="4" sm="3">
+				<v-img
+					style="float:left"		
+					max-height="100"
+					max-width="100" 
+					src="../assets/blankPicture.png">
+				</v-img>
+			</v-col>  
+			<v-col cols="8" sm="6">
+				<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataPersonalInfo.familyName}}, {{dataPersonalInfo.givenName}} {{dataPersonalInfo.suffix}}</span></div>
+				<div class="font-weight-medium">DOB:  <span class="font-weight-regular">{{dataPersonalInfo.birthDate}}</span></div>
+				<div class="font-weight-medium">Gender ID:  <span class="font-weight-regular">{{dataPersonalInfo.gender}}</span></div>
+				<div class="font-weight-medium">Race(s):  <span class="font-weight-regular">{{dataPersonalInfo.raceSelections}}</span></div>
+				<div class="font-weight-medium">Ethnicity:  <span class="font-weight-regular">{{dataPersonalInfo.ethnicitySelection}}</span></div>
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="12" sm="6" md="3">
+				<div class="font-weight-medium primary--text">Home Address </div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Street Address:  <span class="font-weight-regular">{{dataHomeAddress.lineAddress}}</span></div>
+				<div class="font-weight-medium">City, County, State, Country, Zip Code:  <span class="font-weight-regular">{{dataHomeAddress.cityAddress}}, {{dataHomeAddress.districtAddress}}, {{dataHomeAddress.stateAddress}}, {{dataHomeAddress.countryAddress}}, {{dataHomeAddress.postalCode}}</span></div>
+			</v-col>
+		</v-row>
+
+		<v-row>
+			<v-col cols="12" sm="6" md="3">
+				<div class="font-weight-medium primary--text"> Contact Information</div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Phone: <span class="font-weight-regular">{{dataContactInfo.phoneNumber}} ({{dataContactInfo.phoneNumberType}})</span></div>
+				<div class="font-weight-medium">E-mail: <span class="font-weight-regular">{{dataContactInfo.email}}</span></div>
+				<div class="font-weight-medium">Follow-up approval: <span class="font-weight-regular">{{dataContactInfo.approval}}</span></div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12" sm="6" md="3">
+				<div class="font-weight-medium primary--text"> Emergency Contact </div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataEmergencyContact.contactFamilyName}}, {{dataEmergencyContact.contactGivenName}}</span></div>
+				<div class="font-weight-medium">Phone:  <span class="font-weight-regular">{{dataEmergencyContact.contactPhoneNumber}} ({{dataEmergencyContact.contactPhoneNumberType}})</span></div>
+			</v-col>
+		</v-row>
 	</v-container>
 </template>
  
 <script>
+import EventBus from '../eventBus'
+
 	export default {
 	data () {
 		return {
-			
-
+			dataPersonalInfo:
```

Comment:
Yea going back to how will we handle household information in this framework? Will variable names change and we fit in here or is there another event bus?

Reply:
We can use the same strategy (event bus), but the data will be forwarded to a separate page (review and submit page for the "household" registration).